src/extensions/surveys.tsx

```diff
@@ -759,7 +759,7 @@ export function Questions({
         survey.appearance?.backgroundColor || defaultSurveyAppearance.backgroundColor
     )
     const [questionsResponses, setQuestionsResponses] = useState({})
-    const { isPreviewMode, previewPageIndex, onPopupSurveyDismissed, isPopup, onPreviewSubmit, onPopupSurveySent } =
```

Comment:
don we need the `isPreviewMode` anymore?

Reply:
not on this part anymore, only to prevent the submission now (another component)